test/test_new_big_phi.py

```diff
@@ -3,6 +3,7 @@
 # test_new_big_phi.py
 
 import json
+from pyrsistent import v
```

Comment:
I think this is a VSCode Intellisense addition that was unintentional? Otherwise, LGTM!